diff --git a/rt-thread/src/idle.c b/rt-thread/src/idle.c
index 02a234b..2a6e669 100644
--- a/rt-thread/src/idle.c
+++ b/rt-thread/src/idle.c
@@ -211,7 +211,7 @@ void rt_thread_idle_init(void)
 {
     /* initialize thread */
     rt_thread_init(&idle,
-                   "tidle",
+                   "tidle0",
                    rt_thread_idle_entry,
                    RT_NULL,
                    &rt_thread_stack[0],
