diff --git a/rt-thread/components/finsh/finsh_config.h b/rt-thread/components/finsh/finsh_config.h
index 2561e20..2362af5 100644
--- a/rt-thread/components/finsh/finsh_config.h
+++ b/rt-thread/components/finsh/finsh_config.h
@@ -11,11 +11,15 @@
 // <o>the priority of finsh thread <1-30>
 //  <i>the priority of finsh thread
 //  <i>Default: 21
+#ifndef FINSH_THREAD_PRIORITY
 #define FINSH_THREAD_PRIORITY       21
+#endif
 // <o>the stack of finsh thread <1-4096>
 //  <i>the stack of finsh thread
 //  <i>Default: 4096  (4096Byte)
+#ifndef FINSH_THREAD_STACK_SIZE
 #define FINSH_THREAD_STACK_SIZE     1024
+#endif
 
 #define FINSH_USING_SYMTAB
 // <c1>Enable command description
